kyuubi-server/src/main/scala/org/apache/kyuubi/operation/ExecuteStatement.scala

```diff
@@ -81,6 +81,9 @@ class ExecuteStatement(
       while (!isComplete) {
         fetchQueryLog()
         verifyTStatus(statusResp.getStatus)
+        if (statusResp.getProgressUpdateResponse != null) {
+          setOperationJobProgress(statusResp.getProgressUpdateResponse)
```

Comment:
https://github.com/apache/incubator-kyuubi/pull/2529#issuecomment-1120181210 here the best place we stringify the ProgressUpdateResponse and do the log

And maybe refine L88 together

Reply:
SGTM, I will send a follow-up pr to implement it.